fusion/dsl/qparser.cpp

```diff
@@ -0,0 +1,128 @@
+// HEADER PLACEHOLDER
+// contact Jeff Nye, jeffnye-gh
+//
+//! \file qparser.cpp  wrapper around parser state machine(s)
+#include "qparser.h"
+#include <iomanip>
+using namespace std;
+
+extern FILE* yyin;
+extern int yyparse();
+extern QParser* PR;
+
+// ----------------------------------------------------------------
+// ----------------------------------------------------------------
+QParser::QParser() :
+    TRACE_EN(0),
+    lineNo(1),
+    curCol(1),
+    currentFile(""),
+    syntaxName(""),
+    inputFiles()
```

Comment:
You could move these to inline initialization in the header and just drop the ctor altogether.